interface/i18n/mist.it.i18n.json

```diff
@@ -90,10 +90,11 @@
             "blockReceivedShort": "Nuovo blocco",
             "blockNumber": "Il tuo ultimo numero di blocco",
             "timeSinceBlock": "Tempo trascorso dall'ultimo blocco",
-            "testnetExplain": "Sei connesso alla testnet, NON INVIARE alcun ether a questi indirizzi",
+            "testnetExplain": "Sei connesso alla testnet, NON INVIARE alcun ether reale a questi indirizzi",
             "peers": "nodi",
-            "checkingWhichNetwork": "Checking network...",
-            "privateNetwork": "Private-net"
+            "checkingWhichNetwork": "Controllo rete...",
+            "privateNetwork": "Private-net",
+
```

Comment:
Here too, trailing comma and newline need to be removed.

(In the future, you can also try this tool https://jsonlint.com)

Reply:
sorry but that seems a merge error... some lines are missing after the merge!

Reply:
i can confirm that version 8171677 had other lines after the comma, so are these merge errors? can revert to 8171677 version?